File: lesson4/c_c++/homework/homework4_25.cpp
//: lesson4/c_c++/homework/homework4_25.cpp
#include <iostream>

int main() {
    int numb;
    std::cout << "Введите натуральное число => ";
    std::cin >> numb;
    int fact = 1;
    for (int i = 1; i <= numb; i++) {
        fact *= i;
    }
    std::cout << "factorial => " << fact << '\n';
    return 0;
}

File: lesson4/python3/homework/homework4_25.py
#!/usr/bin/python3
#: lesson4/python3/homework/homework4_25.py

numb = int(input("Введите натуральное число => "))

fact = 1
i = 1
while i <= numb:
    fact *= i
    i += 1
print("factorial =>", fact)
